swan-cern/scripts/11_htcondor.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Author: Danilo Piparo, Enric Tejedor, Pedro Maximino 2023
+# Copyright CERN
+# The HT Condor environment is condfigured here.
+
+# HTCondor at CERN integration
+if [[ $CERN_HTCONDOR ]]
+then
+  export CONDOR_CONFIG=/eos/project/l/lxbatch/public/config-condor-swan/condor_config
+  mkdir -p /etc/condor/config.d/ /etc/myschedd/
+  ln -s /eos/project/l/lxbatch/public/config-condor-swan/config.d/10_cernsubmit.erb /etc/condor/config.d/10_cernsubmit.erb
+  ln -s /eos/project/l/lxbatch/public/config-condor-swan/myschedd.yaml /etc/myschedd/myschedd.yaml
+  ln -s /eos/project/l/lxbatch/public/config-condor-swan/ngbauth-submit /etc/sysconfig/ngbauth-submit
+
+  # Create self-signed certificate for Dask processes
+  log_info "Generating certificate for Dask"
+  export DASK_TLS_DIR=$DASK_DIR/tls
+  mkdir $DASK_TLS_DIR
+  chown -R $NB_USER:$NB_USER $DASK_TLS_DIR
+  sudo -u $NB_USER sh /srv/singleuser/create_dask_certs.sh $DASK_TLS_DIR &
+
+  # Dask config: lab extension must use SwanHTCondorCluster
+  DASK_CONFIG_DIR=/etc/dask 
+  mkdir $DASK_CONFIG_DIR
+  echo "
```

Comment:
This could be transformed into a static configuration file that we just copy into `DASK_CONFIG_DIR`.

Reply:
Changed to a file that is moved to the container now